docs/glossary.md

```diff
@@ -33,6 +33,7 @@ The following is a glossary of domain specific terminology. Although benchmarks
 
 ## Analysis
 
+* **artifact comparisons**: the comparison of two artifacts. This is composed of many test result comparisons. The [comparison page](https://perf.rust-lang.org/compare.html) shows a single artifact comparison between two artifacts.
```

Comment:
So is "artifact" equivalent to "rustc (and/or rustdoc) version"?

Reply:
Depends on what you mean by "version". Specifically, an artifact is some rustc/rustdoc binary that is labeled by some identifier. The identifier can be a version number (e.g., 1.60.0), a git hash, or in general any arbitrary string (e.g., "test1").

Reply:
This extra info about the rustc/rustdoc binary would be good to add.